### DISCUSSION TITLE
导出笔记时如何取消笔记链接？

### DISCUSSION THREAD

**Discussion by Tcould** [windingwind/zotero-better-notes#1149] [Q&A]
导出笔记时如何取消笔记链接？当我导出笔记时笔记的链接模式已经自动选择了并且无法取消，有什么办法取消笔记链接吗？
![image](https://github.com/user-attachments/assets/9dd02c59-20cb-4d09-8e54-8c3d8eceed37)

**Comment by windingwind** [comment]
是指删除所有笔记链接？没有这个选项

**Reply by Tcould** [reply]
就是在笔记链接模式此处，是否可以取消勾选，不选择笔记链接。
![screenshots](https://github.com/user-attachments/assets/e46f11b7-72ef-42fd-9cb2-e01a43ef3cd8)

**Reply by windingwind** [reply]
请描述你希望导出以后的内容长什么样

**Reply by Tcould** [reply]
所导出笔记中的注释不添加上回链，仅仅是文本。

**Reply by Tcould** [reply]
刚刚经过摸索发现，运用系统自带的导出就可以导出不带回链的注释笔记，感谢作者大大的解答，有同样需求的朋友也可以这样试下
![screenshots](https://github.com/user-attachments/assets/a49881a3-83f8-43f4-a632-2e27d6f9775e)
